dbms/src/Storages/Transaction/tests/gtest_kvstore.cpp

```diff
@@ -1092,22 +1058,85 @@ void RegionKVStoreTest::testKVStore()
             catch (Exception & e)
             {
                 ASSERT_EQ(e.message(), "[region 19] already has newer apply-index 8 than 6, should not happen");
-                ASSERT_EQ(kvs.getRegion(19)->dataInfo(), "[default 2 ]"); // apply-snapshot do not work
             }
-            kvs.handleApplySnapshot(
-                region->getMetaRegion(),
-                2,
-                {}, // empty
-                8, // same index
-                5,
-                ctx.getTMTContext());
-            ASSERT_EQ(kvs.getRegion(19)->dataInfo(), "[default 2 ]"); // apply-snapshot do not work
-            region = makeRegion(19, RecordKVFormat::genKey(1, 50), RecordKVFormat::genKey(1, 60));
-            region->handleWriteRaftCmd({}, 10, 10, ctx.getTMTContext());
-            kvs.checkAndApplySnapshot<RegionPtrWithBlock>(region, ctx.getTMTContext());
-            ASSERT_EQ(kvs.getRegion(19)->dataInfo(), "[]");
+        }
+
+        {
+            {
+                auto region = makeRegion(22, RecordKVFormat::genKey(55, 95), RecordKVFormat::genKey(65, 95));
```

Comment:
```suggestion
                auto region = makeRegion(22, RecordKVFormat::genKey(55, 95), RecordKVFormat::genKey(55, 1000));
```
Now the key range of region synced to TiFlash only contains one table id. We should call `makeRegion` with key range of the same table id.